packages/shared/src/lib/rank.ts

```diff
@@ -2,6 +2,74 @@ export const NO_RANK = 'No rank';
 export const FINAL_RANK = 'You made it 🎉';
 export const STEPS_PER_RANK = [3, 4, 5, 6, 7];
 export const RANK_NAMES = ['Bronze', 'Silver', 'Gold', 'Platinum', 'Diamond'];
```

Comment:
We probably need to update this as well or the areas that implement this to utilize the new one. What do you think?

Reply:
Yep we'll use the new RANKS object for this as well, wanted to take this on in DD-507 (Aya's ticket)

Reply:
Actually @sshanzel decided to add this in this PR :)
Feel free to check again